clients/unshaded/src/main/java/tachyon/client/TachyonFS.java

```diff
@@ -140,7 +140,7 @@ public static synchronized TachyonFS get(String masterHost, int masterPort, bool
    * @return the corresponding TachyonFS handler
    */
   public static synchronized TachyonFS get(TachyonConf tachyonConf) {
-    Preconditions.checkArgument(tachyonConf != null, "Could not pass null TachyonConf instance.");
+    Preconditions.checkArgument(tachyonConf != null, "Could not pass a null TachyonConf instance.");
```

Comment:
Could you change the comments to be like `TachyonConf cannot be null.` to be consistent with the other messages? (Same comment for the check above)